repl.fnl

```diff
@@ -0,0 +1,45 @@
+(local coroutine (require :coroutine))
+(local fennel (require :fennel))
+(local jeejah (require :jeejah))
+(local view (require :fennelview))
+
+(fn fennel-middleware
```

Comment:
Could you please add some docstrings to the module with an example and functions?